index.js

```diff
@@ -10,36 +10,39 @@ module.exports = {
     Request: true
   },
   rules: {
-    'arrow-parens': ['error', 'as-needed'],
     'arrow-body-style': 'off',
+    'arrow-parens': ['error', 'as-needed'],
     'comma-dangle': ['error', 'never'],
     'consistent-return': 'off',
     'default-case': 'off',
     'func-names': 'off',
+    'function-paren-newline': ['error', 'consistent'],
     'generator-star-spacing': ['error', { before: true, after: true }],
+    'import/no-dynamic-require': 'off',
+    indent: ['error', 2, {
+      MemberExpression: 'off',
+      SwitchCase: 1
+    }],
+    'no-cond-assign': ['error', 'except-parens'],
     'no-confusing-arrow': 'off',
     'no-continue': 'off',
+    'no-multi-spaces': ['error', { ignoreEOLComments: true }],
     'no-param-reassign': 'off',
     'no-path-concat': 'off',
     'no-plusplus': 'off',
     'no-restricted-properties': 'off',
     'no-return-assign': 'off',
+    'no-return-await': 'off',
```

Comment:
I started to abide this rule on our code, but I'm not satisfied with it completely. Particularly in these cases:

```javascript
const someFun = async things => aSyncMethod() && await anAsyncMethod(things);
```

This becomes an error because technically the `await` is redundant. However, it makes explicit which of the calls is synchronous and which is not, therefore making the code clearer. So I'm removing the rule.